draft-ietf-tls-tls13.md

```diff
@@ -3619,24 +3613,22 @@ The traffic keying material is generated from the following input values:
   being generated for.
 * A purpose value indicating the specific value being generated
 * The length of the key.
-* The handshake context which is used to generate the keys.
 
 The keying material is computed using:
 
        key = HKDF-Expand-Label(Secret,
-                               phase + ", " + purpose,
-                               handshake_context,
+                               phase + ", " + purpose, "",
                                key_length)
 
 The following table describes the inputs to the key calculation for
 each class of traffic keys:
 
-| Record Type | Secret | Phase | Handshake Context |
-|:------------|--------|-------|------------------:|
-| 0-RTT Handshake   | xSS | "early handshake key expansion" | ClientHello  |
-| 0-RTT Application | xSS | "early application data key expansion" | ClientHello |
-| Handshake         | xES | "handshake key expansion" | ClientHello... ServerHello |
-| Application Data  | traffic secret | "application data key expansion" | ClientHello... Server Finished |
+| Record Type | Secret | Phase | 
+|:------------|--------|-------|
+| 0-RTT Handshake   | early_traffic_secret | "early handshake key expansion" |
+| 0-RTT Application | early_traffic_secret | "early application data key expansion" | 
+| Handshake         | handshake_traffic_secret | "handshake key expansion" |
+| Application Data  | traffic secret_N | "application data key expansion" | 
```

Comment:
missing underscore here; should be "traffic_secret_N"

(noted previously, but comment appears to have gotten lost on GitHub for some reason)